src/Views/ViewStack.vala

```diff
@@ -39,6 +41,9 @@ public class Noise.ViewStack : Gtk.Stack {
                 update_visible ();
             }
         });
+
+        var alert_view = new Granite.Widgets.AlertView ("No results", "Try another search", "edit-find-symbolic");
```

Comment:
Make sure that these strings are translatable :) Titles should be in title case and descriptions should be sentence case with terminating punctuation. In Switchboard, for example, the strings are:

_("No Results"),
_("Try changing search terms.")